tools/cldr-apps/js-unittest/TestCldrStForumFilter.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
-const assert = chai.assert;
+var assert = chai.assert;
```

Comment:
why const to var?

Reply:
It's in the global namespace, and it's declared in two files (so far): TestCldrStForum.js and TestCldrStForumFilter.js. If they both used const, we'd get an error. That's why I changed it when adding the second test file. Now I think about it more, one alternative would be to add brackets around everything and add a level of indentation. Then const would be OK and we'd avoid polluting the global namespace. Maybe I'll do that the next time I revise those files.

Reply:
That makes sense about global scope.